pyproject.toml

```
@@ -39,3 +39,39 @@ skip_glob = [
     "mypyc/test-data/*",
     "test-data/*",
 ]
+
+[tool.ruff]
+line-length = 99
+target-version = "py37"
+
+select = [
+  "E",       # pycoderstyle (error)
+  "F",       # pyflakes
+  "B",       # flake8-bugbear
+  "RUF100",  # flake8-noqa
```

PGH004 corresponds to the `noqa_require_code` setting for flake8-noqa that we currently have enabled

```suggestion
  "RUF100",  # Unused noqa comments
  "PGH004"  # blanket noqa comments
```